.gitlab-ci.yml

```diff
@@ -10,27 +11,49 @@ variables:
   LIVE_DOMAIN: "https://docs.datadoghq.com/"
   PREVIEW_DOMAIN: "https://docs-staging.datadoghq.com/"
   FORCE_COLOR: '1'
+  GIT_DEPTH: 50
```

Comment:
might need to be careful using git_depth on all jobs and check them. it tends to cause finding tags or comparing file changes with git commands to not always work.